src/pages/index.astro

```diff
@@ -2,241 +2,177 @@
-      </div>
-    </section>
-  )}
-  <section class="section">
-    <div class="contentWrapperLg">
-      <div class="sectionHeader">
-        <h2 class="heading3">Tools</h2>
-      </div>
-      <div class="threeColGrid toolsList">
-        <a class="gridItem" href="/play" data-umami-event="Landing page - Playground">
-          <h3 class="heading4">Web Monetization Playground</h3>
-          <p>Explore how Web Monetization works</p>
-        </a>
-
-        <a class="gridItem" href="/tools/link-tag" data-umami-event="Landing page - Link tag">
-          <h3 class="heading4">Link Tag Generator</h3>
-          <p>Generate a Web Monetization link tag for your site</p>
-        </a>
-
-        <a class="gridItem" href="/tools/prob-revshare" data-umami-event="Landing page - Prob rev share">
-          <h3 class="heading4">Probabilistic RevShare Generator</h3>
-          <p>Split Web Monetization revenue between wallet addresses</p>
-        </a>
-      </div>
-    </div>
-  </section>
-  <section class="section">
-    <div class="contentWrapperLg">
-      <div class="sectionHeader">
-        <h2 class="heading3">Contributors</h2>
-        <p>Thank you to all our contributors.</p>
-      </div>
-      {contributors.length > 0 && (
-        <ul class="contributors">
-          {contributors.map((contributor) => (
-            <li class="contributor">
-              <a href={contributor.html_url}>
-                <img alt={contributor.login} src={contributor.avatar_url} />
-              </a>
-            </li>
-          ))}
-        </ul>
       )}
+      </div>
     </div>
   </section>
 </Base>
 
 <style>
-/* Hero section styles */
-.hero {
-  background-image: url("/img/bg-lines--light.svg"), url("/img/bg-tile.svg");
-  background-repeat: repeat-x, repeat;
-  background-size: 17em, 20em;
-  background-position: bottom, center;
-  margin-block-end: var(--space-xl);
+section.hero {
   text-align: center;
-}
-.heroIcon {
-  width: 120px;
-  margin-block-end: var(--space-s);
-  margin-inline: auto;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
 }
 
-.hero h1 {
-  color: var(--wm-accent);
+.hero .contentWrapper {
+  height: 100%;
+  display: flex;
+  flex-grow: 1;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
 }
 
-/* Card wrapper styles */
-.cards {
+.heading-wrapper {
   display: flex;
-  flex-wrap: wrap;
-  gap: var(--space-s);
-  margin-block-end: var(--space-s);
-  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: var(--spacing-lg);
 }
 
-/* Tools list styles */
-.toolsList {
-  justify-content: center;
+.heading-wrapper h1 {
+  margin: 0;
 }
 
-.toolsList a {
-  text-align: center;
-  cursor: pointer;
+.heading-wrapper img {
+  width: 75%;
+  max-width: 25rem;
+  margin-top: 0.5rem;
+  height: auto;
+}
+
+.hero-video {
+  flex: 1 1 100%;
+  max-height: 50vh;
+  max-width: 80vw;
+  width: auto;
+  object-fit: contain;
```

Comment:
This should help prevent a layout shift.
```suggestion
  max-height: 50vh;
  max-width: 80vw;
  object-fit: contain;
  aspect-ratio: 1280 / 941;
```

https://github.com/user-attachments/assets/94a22c6b-d322-47d9-9460-2ca4209976e1